src/Tizen.TV.UIControls.Forms/Focus.cs

```diff
@@ -182,27 +182,27 @@ static void KeyHandler(RemoteControlKeyEventArgs args)
         }
     }
 
-    class PlatformFocusUpEffect : RoutingEffect
+    public class FocusUpEffect : RoutingEffect
```

Comment:
Is there any reason to change to public?

Reply:
Thank you for your suggestion.
I agree that it is better to separate the file for each public classes and I've fixed it.